### DISCUSSION TITLE
Comparing large files as text via CLI

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [WinMerge/winmerge#1512] [Q&A] [answered]
I have 2 large text files, each 155MB.  I would like to compare them as text to see the differences.

When I use WinMerge via its CLI to compare these 2 files, WinMerge just tells me the files are different.  I would like to see the changes like WinMerge does for smaller filers.

How can I set WinMerge to do this?

**Comment by Gitoffthelawn** [comment]
As a temporary workaround that may help others, I found that this WinMerge fork works well for this task: https://github.com/datadiode/winmerge2011

**Comment by sdottaka** [comment, marked as answer]
In the Options dialog, if the checkbox is checked as shown below, please uncheck it.

![image](https://user-images.githubusercontent.com/98126/194876604-797de17d-6445-4513-a688-6fe99341e795.png)

**Reply by Gitoffthelawn** [reply]
Ah, that should do it.  Thank you Takashi!
